### PR TITLE
Add postgres lib to create servers.

### PR DESCRIPTION
**Pull request by vitorarins**
Azure SDK for Go still doesn't have the package necessary to use PostgreSQL managed Databases. So there are no tests for now :disappointed:

Any suggestions are welcome :smiley:

### CODE REVIEW

**Comment by katcipis** [comment]
Well you could talk directly through the REST API taking a look at the current Go code generated. But I'm not sure if it is worth it, we could just wait for it to became available.

**Comment by katcipis** [comment]
I have another suggestion =)

You could provide a cool example and run it like this:

https://github.com/NeowayLabs/klb/blob/master/tests/azure/examples_test.go#L17

It is not great, but seems better than nothing

File: azure/postgres.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+fn azure_postgres_set_performance_tier(instance, tier) {
+	instance <= append($instance, "--performance-tier")
+	instance <= append($instance, $tier)
+
+	return $instance
+}
+
+# azure_postgres_disable_ssl disables the ssl enforcement.
+# `instance` is the postgres server instance.
+fn azure_postgres_disable_ssl(instance) {
```
Comment (vitorarins) [inline azure/postgres.sh]: I am still in doubt about this functions. Maybe it should be `azure_postgres_set_ssl(instance, ssl)`
@katcipis @lborguetti what do you think?

Comment (katcipis) [inline azure/postgres.sh]: @vitorarins I prefer disable/enable functions, they are more clear, specially because nash does not have booleans.

**Pull request merged**